packages/extensionmanager/src/dialog.tsx

```diff
@@ -19,9 +19,7 @@ export function reportInstallError(
   translator = translator || nullTranslator;
   const trans = translator.load('jupyterlab');
   const entries = [];
-  entries.push(
-    <p>{trans.__('An error occurred installing <code>%1</code>.', name)}</p>
-  );
+  entries.push(<p>{trans.__(`An error occurred installing "${name}."`)}</p>);
```

Comment:
Looks like the dot could be moved:

```suggestion
  entries.push(<p>{trans.__(`An error occurred installing "${name}".`)}</p>);
```